emcc.py

```diff
@@ -18,9 +18,6 @@
                (by default /tmp/emscripten_temp). "2" will save additional emcc-*
                steps, that would normally not be separately produced (so this
                slows down compilation).
-
-  EMMAKEN_NO_SDK - Will tell emcc *not* to use the emscripten headers. Instead
-                   your system headers will be used.
```

Comment:
This is not the same behavior as `-nostdinc`. nostdinc will not use any system headers at all, but this behavior sounds like it will just try to find local ones.

Reply:
Having said that the behavior described here will vary a lot by platform, and I really don't see how it's useful at all.

Reply:
That aspect of this feature seems to have been broken for while.    I think because `clang -target wasm32-unknown-emscripten` won't look in `/usr/include` but default, even without `--sysroot`.  So I think one would need to explicitly add `-I/usr/include` to achieve that.